server/recceiver/cfstore.py

```diff
@@ -71,6 +71,16 @@ def from_config_adapter(conf: ConfigAdapter) -> "CFConfig":
         )
 
 
+@dataclass
+class CFProperty:
+    name: str
+    owner: str
+    value: Optional[str] = None
+
+    def as_dict(self) -> Dict[str, str]:
+        return {"name": self.name, "owner": self.owner, "value": str(self.value)}
```

Comment:
Given `value: Optional` I'd say this should be
```python
    def as_dict(self) -> Dict[str, Optional[str]]:
        return {"name": self.name, "owner": self.owner, "value": str(self.value)}
```
and you're probably better off doing `"value": self.value or ""` (to avoid the string `"None"`)